hudi-flink/src/main/java/org/apache/hudi/sink/StreamWriteOperatorCoordinator.java

```diff
@@ -78,19 +71,19 @@
   private final Configuration conf;
 
   /**
-   * Write client.
+   * Coordinator context.
    */
-  private transient HoodieFlinkWriteClient writeClient;
```

Comment:
these changes are confusing, we deleted the old one and insert a new line(L81)?

Reply:
Yes, i re-organize the variables.